src/main/kotlin/snyk/oss/annotator/OSSNpmAnnotator.kt

```diff
@@ -62,14 +61,21 @@ class OSSNpmAnnotator : OSSBaseAnnotator() {
 
         override fun visitElement(element: PsiElement) {
             if (isSearchedDependency(element)) {
-                val value = element.getNextSiblingIgnoringWhitespace(false)
-                    ?.getNextSiblingIgnoringWhitespace(false) ?: return
+                val value = element.getNextSiblingIgnoringWhitespace()?.getNextSiblingIgnoringWhitespace() ?: return
                 this.foundTextRange = TextRange(element.textRange.startOffset, value.textRange.endOffset)
                 return
             }
             super.visitElement(element)
         }
 
+        private fun PsiElement.getNextSiblingIgnoringWhitespace(): PsiElement? {
+            var candidate = this.nextSibling
+            while (candidate is PsiWhiteSpace) {
+                candidate = candidate.nextSibling
+            }
+            return candidate
+        }
+
```

Comment:
copied from the kotlin-plugin, right?

Reply:
nope, handmade :)
kotlin-plugin is close-source (?) at least I can't see it source...

Reply:
Yeah, they did kind of the same way just more generic and through streams and Iterators